subsys/mgmt/osdp/src/osdp_pd.c

```diff
@@ -226,13 +292,21 @@ static int pd_decode_command(struct osdp_pd *pd, uint8_t *buf, int len)
 		if (len != CMD_ISTAT_DATA_LEN) {
 			break;
 		}
+		if (!pd_cmd_cap_ok(pd, NULL)) {
```

Comment:
Are there any requirements defined for POLL command ?

Reply:
No, all PD have to support the POLL command.

Reply:
So you can remove this capabilities check

Reply:
This check is not in `case CMD_POLL`. It is for `case CMD_ISTAT`.